internal/pkg/java/server.go

```diff
@@ -44,6 +45,14 @@ func (s *Server) SetLogger(log logr.Logger) {
 	s.Log = log
 }
 
+func (s Server) GetSendProxyProtocol() bool {
+	return s.SendProxyProtocol
+}
+
+func (s Server) GetSendRealIP() bool {
+	return s.SendRealIP
+}
+
```

Comment:
You actually don't need to create getters for these attributes since you intend to use them inside of ProcessConn which is a method of Server. The other getters only exist to fulfill the infrared.Server interface